machine-learning/solutions/energy_price_forecasting/README.md

```diff
@@ -71,4 +71,20 @@ gcloud ml-engine jobs submit prediction ${JOB_NAME} \
     --region=us-central1 \
     --data-format=TEXT \
     --version=${MODEL_VERSION}
-```
\ No newline at end of file
+```
+
+#LICENSE
```

Comment:
Put License at the top of each file with source code (markdown does not count, so LICENSE can be removed from this file) https://g3doc.corp.google.com/company/teams/opensource/releasing/preparing.md?cl=head

You can look at tensorflow github repo for examples.